build/templates/session.py.mako

```diff
@@ -124,6 +124,11 @@ init_call_params = helper.get_params_snippet(init_function, helper.ParameterUsag
 <%include file="${'/session.py' + method_template['session_filename'] + '.py.mako'}" args="f=functions[func_name], config=config, method_template=method_template" />\
 % endfor
 % endfor
+% for custom_method in config['custom_methods']:
+% if custom_method['channel_based']:
```

Comment:
We already have a key `render_in_session_base` to denote "generate this in the base class". You should use that. I can see us needing to put hand coded methods in the base class regardless of whether they are channel based.